.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,30 @@
+## Description ##
+<!--- Tell us what this pull request does in the most simple to understand way  -->
+
+## Type of change ##
+Please select the relevant option
+- [ ] Bug fix(a non-breaking change which fixes an issue)
+- [ ] New feature(a non-breaking change which adds functionality)
+- [ ] Breaking change(fix of a feature that would cause existing functionality to not work as expected)
+- [ ] This change requires a documentation update
+
+## How Has This Been Tested? ##
+Please describe the tests that you ran to verify your changes
+- [ ] End to end test
+- [ ] Integration test
+- [ ] unit test
+
+## Checklist: ##
+- [ ] My code follows the style guidelines of this project
+- [ ] I have linted my code prior to submission
+- [ ] I have made corresponding changes to the documentation
+- [ ] My changes generate no new warnings
+- [ ] I have added tests that prove my fix is effective or that my feature works
+- [ ] New and existing tests pass locally with my changes
+
+
+## How can this PR be manually tested: ##
+<!-- Please describe steps to successfully test this work -->
+
```

Comment:
Good work @CryceTruly . However, I think it would be good to have a section where someone raising a PR can leave the information on the background context while testing the PR. This can be any useful information or environment valiable that is required while testing the PR.

Reply:
Okay @shemaeric ,  will look into it.

Reply:
Hey @shemaeric, I have worked on this, Kindly check again.